### PR TITLE
[opentelemetry] test values, bump chart

### PR DESCRIPTION
**Pull request by Kuckkuck**
Adding ci folder for testing
moving a lot of defaults to test-values.yaml
adding missing defaults.
labels for prometheus

### CODE REVIEW

**Comment by ibakshay** [comment]
@Kuckkuck, I cannot see the ` test-values.yaml` in the ci folder. Did you forget to push this change?

File: opentelemetry/chart/templates/logs-collector.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ spec:
       port: 9999
 {{- end }}
   image: {{ index .Values "opentelemetry-operator" "manager" "collectorImage" "repository" }}:{{ index .Values "opentelemetry-operator" "manager" "collectorImage" "tag" }}
+  upgradeStrategy: automatic
```
Comment (viennaa) [inline opentelemetry/chart/templates/logs-collector.yaml:42]: is this really needed? it reads as if this is the default

https://pkg.go.dev/github.com/open-telemetry/opentelemetry-operator#section-readme

> The default and only other acceptable value for .Spec.UpgradeStrategy is automatic.

File: opentelemetry/chart/templates/smon.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ metadata:
   annotations:
   name: opentelemetry-collector-metrics
   labels:
+    plugin: "{{ .Values.openTelemetry.prometheus }}"
```
Comment (viennaa) [inline opentelemetry/chart/templates/smon.yaml:12]: How does openTelemetry.prometheus map to the otel plugin name?

**Review by viennaa — COMMENTED** [review]
Thanks for updating the chart. Please check my remarks.

File: opentelemetry/chart/templates/smon.yaml (hunk 2 above)
Comment (Kuckkuck) [reply opentelemetry/chart/templates/smon.yaml:12]: at the moment it has to be set in the plugin page.

**Review by ibakshay — COMMENTED** [review]
The helm test is passing now! 🚀

**Review by viennaa — CHANGES_REQUESTED** [review]
👍 for including and using the label helper.

can you integrate the functionality to add custom labels too, please?

example:
https://github.com/cloudoperators/greenhouse-extensions/blob/main/thanos/charts/templates/store-svc.yaml#L14-L16

**Review by viennaa — APPROVED** [review]
Thanks! Go.

**Pull request merged**